packages/@expo/metro-config/src/ExpoMetroConfig.ts

```diff
@@ -244,6 +244,10 @@ export function getDefaultConfig(
       unstable_allowRequireContext: true,
       allowOptionalDependencies: true,
       babelTransformerPath: require.resolve('./babel-transformer'),
+      // See: https://github.com/facebook/react-native/blob/v0.73.0/packages/metro-config/index.js#L72-L74
+      asyncRequireModulePath: resolveFrom.silent(projectRoot, '@expo/metro-runtime')
+        ? require.resolve('@expo/metro-runtime/async-require')
```

Comment:
These could resolve to two different copies, resolve once and use the results of `resolveFrom`.